web-compose/src/jsMain/kotlin/BrowserViewportWindow.kt

```diff
@@ -0,0 +1,80 @@
+@file:Suppress(
+    "INVISIBLE_MEMBER",
+    "INVISIBLE_REFERENCE",
+    "EXPOSED_PARAMETER_TYPE"
+) // WORKAROUND: ComposeWindow and ComposeLayer are internal
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.window.ComposeWindow
+import kotlinx.browser.document
+import kotlinx.browser.window
+import org.w3c.dom.HTMLCanvasElement
+import org.w3c.dom.HTMLStyleElement
+import org.w3c.dom.HTMLTitleElement
+
+private const val CANVAS_ELEMENT_ID = "ComposeTarget" // Hardwired into ComposeWindow
+
+/**
+ * A Skiko/Canvas-based top-level window using the browser's entire viewport. Supports resizing.
+ * Taken from: https://kotlinlang.slack.com/archives/C01F2HV7868/p1660083398571449
+ */
+fun BrowserViewportWindow(
```

Comment:
Using this wrapper from [Oliver on Slack](https://kotlinlang.slack.com/archives/C01F2HV7868/p1660083398571449) instead of `CanvasBasedWindow` because it plays nicer with web browsers and resizing of the window.